### ISSUE TITLE
Focus does not circle back into sim (Safari only and outdated MacOS)

### ISSUE THREAD

**Issue by terracoda** [phetsims/john-travoltage#402]
In https://github.com/phetsims/john-travoltage/issues/400#issuecomment-828760841 I noticed that when I tab out of the dialog and get into the browser controls, I can NOT seem to get back into the sim with Tab key presses.

This actually only happens in Safari. I circle around in FF and Chrome.
It might be because I have an outdated system, too, Mac OS 10.13.6

I don't think this is an issue for visual users who can use a mouse. They can click on the "X". 

My concern is for keyboard-only users and blind users could get really lost as the browser controls won't being voicing.

I don't know if this is actually a Safari bug. 
@jessegreenberg can you or QA test with latest MacOS.

**Comment by jessegreenberg** [comment]
Interesting, I am not sure why this is happening. But it is probably outside the scope of our simulation and a Safari behavior because we aren't using any JavaScript to control focus in and out of the sim.

I just tested on macOS 11.2 with Safari 14.0.3 and did not see this happen. I just tabbed around the browser controls and then landed on the "Close" button back in the sim.

I will mention this to QA when they test and see if they can get this to happen.
